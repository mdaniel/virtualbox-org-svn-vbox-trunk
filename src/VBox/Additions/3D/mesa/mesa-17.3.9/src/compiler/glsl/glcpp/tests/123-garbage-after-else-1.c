#if 1
#else garbage
#endif
