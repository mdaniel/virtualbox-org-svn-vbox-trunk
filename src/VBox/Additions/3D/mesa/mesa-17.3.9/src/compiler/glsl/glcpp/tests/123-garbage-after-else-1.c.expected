0:2(7): preprocessor error: syntax error, unexpected IDENTIFIER, expecting NEWLINE
0:1(6): preprocessor error: Unterminated #if


